lib/tasks/rubocop.rake

```diff
@@ -0,0 +1,3 @@
+require "rubocop/rake_task"
+
+RuboCop::RakeTask.new
```

Comment:
I'd prefer we inline this in the `Rakefile` because putting this in `lib` implies that it is meant to be packaged and distributed as part of the raygun gem.